istioctl/cmd/istioctl/inject.go

```diff
@@ -306,6 +308,9 @@ func init() {
 	injectCmd.PersistentFlags().StringVar(&includeIPRanges, "includeIPRanges", "",
 		"Comma separated list of IP ranges in CIDR form. If set, only redirect outbound "+
 			"traffic to Envoy for IP ranges. Otherwise all outbound traffic is redirected")
+	injectCmd.PersistentFlags().StringVar(&includeInboundPorts, "includeInboundPorts", inject.DefaultIncludeInboundPorts,
+		"Comma separated list of inbound ports for which traffic is to be redirected to Envoy. "+
+			"All ports can be redirected with the wildcard character *.")
```

Comment:
let's treat "" also as same as before, ie intercept everything; it will help upgrades/backward compatibility
maybe have "NONE" as the intercept nothing marker